.travis.yml

```diff
@@ -96,9 +95,9 @@ deploy:
   access_key_id: ${ACCESS_KEY_ID}
   secret_access_key: ${SECRET_ACCESS_KEY}
   bucket: "beam-wheels-staging"
-  skip_cleanup: true
```

Comment:
why was this true?

Reply:
`skip_cleanup` is deprecated and it's replacement is `cleanup`

I should have provided a list of warning messages from https://config.travis-ci.com/explore